src/libsync/discovery.cpp

```diff
@@ -1666,6 +1666,9 @@ void ProcessDirectoryJob::processFileFinalize(
         ASSERT(_dirItem && _dirItem->_instruction == CSYNC_INSTRUCTION_RENAME);
         // This is because otherwise subitems are not updated!  (ideally renaming a directory could
         // update the database for all items!  See PropagateDirectory::slotSubJobsFinished)
+        auto adjustedOriginalPath = _discoveryData->adjustRenamedPath(path._original, SyncFileItem::Down);
```

Comment:
@mgallien is this a leftover?

Reply:
this is needed to avoid having an error due to `[[nodiscard]]`